packages/android_intent/lib/android_intent.dart

```diff
@@ -152,6 +160,26 @@ class AndroidIntent {
     );
   }
 
+  /// Show the default selection dialog to allow the user to select
+  /// an application to resolve this intent from the list of available applications.
+  ///
+  /// [action] must be specified.
+  ///
+  /// [package] and [componentName] will be ignored.
+  ///
+  /// This works only on Android platforms.
+  Future<void> showChooser() async {
```

Comment:
I'll connect my earlier question to this point here. When a developer invokes this method, does that developer expect that a chooser will definitely be displayed? If so, how can the developer take action in the case that no `Activity` is available, or in the case that no target `Activity`s exist?